confluent_kafka/src/confluent_kafka.c

```diff
@@ -1698,6 +1698,9 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
 		char errstr[256];
                 int r = 0;
 
+                if (vo == Py_None)
```

Comment:
Not sure this is the right thing to do since any property with a None value will be silently ignored, which might not be what is expected.

E.g., someone does this:
```
{ ..
    'some.java.property.we.dont.have': some_var_that_happens_to_be_None
}
```

Suggest setting the callbacks explicitly as needed instead.

Reply:
Setting it to None is pretty explicit. I'd argue it expresses enough intent to safely ignore.

Reply:
Explicitly to do what?
I think it is ambigious, does it mean to revert to default? Set it to an empty/zero value depending on type? Better to error out if None is provided.

Reply:
Also see the existing None checks below.

Reply:
This has not been adressed.